src/themes/bankrate/theme.json

```diff
@@ -219,26 +221,12 @@
     }
   },
 
-  "elements": {
-    "list": {
-      "bullet": {
-        "color": "grey-80"
-      },
-      "numeric": {
-        "color": "grey-80",
-        "::before": {
-          "content": "\" - \" counter(li)",
-          "left": "-2em",
-          "width": "2.5em"
-        }
-      }
-    }
-  },
-
```

Comment:
This needs adding back in and the deprecated top level list being removed.

Reply:
like this? https://github.com/uswitch/trustyle/pull/596/commits/acae2369566984e261b8ecdf9200f43d65e74c7c

Reply:
Why remove the before content as well? Do we not want that for the theme?

Reply:
I was just copying across what we had donne in the glaceon repo, but I think that the :before stuff lives in the component for all brands